### DISCUSSION TITLE
Changing the extensions registry

### DISCUSSION THREAD

**Discussion by riexn** [gitpod-io/openvscode-server#283] [Q&A]
Would it be possible to change the extensions registry?
My use case is working in an offline environment, and I would want openvscode-server to use [my own openvsx registry](https://github.com/eclipse/openvsx)

**Comment by akosyakov** [comment]
I think you will need to file a request in the upstream for that. @jeanp413 correct me if I'm wrong

**Comment by jeanp413** [comment]
Not possible after compilation as it is inlined but you could:
1. Manually change it [here](https://github.com/gitpod-io/openvscode-server/blob/fee39f3e03be5fc810441db35f0fd5dfb83cdbfd/product.json#L590-L596) but you'll need to build openvscode-server yourself `yarn gulp vscode-reh-web-linux-x64-min`
2. In the docker file, do a search and replace (grep/sed) of the openvsx url with your extension registry url
